owncloudApp/src/main/res/layout/preview_audio_fragment.xml

```diff
@@ -29,32 +29,51 @@
 
     <FrameLayout
         android:id="@+id/visual_area"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="0dp"
-        android:layout_alignParentTop="true"
         android:layout_above="@+id/media_controller"
-        >
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:padding="40dp"
+        app:layout_constraintDimensionRatio="H,1:1"
```

Comment:
If we reduce the padding to 16dp (Same as media_controller) and we move that `layout_constraintDimensionRatio="H,1:1"` to `layout_constraintDimensionRatio="1:1"` we would have the same behavior in landscape and we could remove the specific `preview_audio_fragment` for landscape. What do you think?

Reply:
I didn't know I could leave out the `H` parameter.
However I don't like `16dp` on portrait, as the cover looks to huge:
|16dp|40dp|
|---|---|
|![bla](https://user-images.githubusercontent.com/1891273/114011273-57609b80-9865-11eb-9392-8a97dc984067.png)|![asdf](https://user-images.githubusercontent.com/1891273/114012387-8deae600-9866-11eb-85c2-faf2e8cd6b60.png)|

Reply:
All right I've settled with 30dp for land/portrait. :)